server/etc/e-smith/templates/etc/dovecot/quota.passwd/10base

```diff
@@ -3,6 +3,9 @@
 #
 {
     use esmith::AccountsDB;
+    my $db = esmith::ConfigDB->open_ro();
+    my $domain = $db->get_value('DomainName');
```

Comment:
You could avoid getting DomainName from DB by discarding everything after the `@` character.

Reply:
Actually you're opening the database twice and this should be avoided.

I'd remove lines from 6 to 8.